status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.174.6",
-    "commit-sha1": "f95dd35d131f68645b7c4dc5fbcea955c16e25bd",
-    "src-sha256": "188chrifz4h3j6fpsrqq83v2vcnpp8y4282k9lhlrkkknfgwz5v2"
+    "version": "b1da689ca06dfc66d47f49beb457b564e2fe6fea",
```

Comment:
do we need a new status-go version for this work?

Reply:
@J-Son89 for this PR not needed, the feature is still hidden behind feature flag

Reply:
@J-Son89 updated status-go version